### PR TITLE
SG-22663: Shotgun to ShotGrid

### PR DESCRIPTION
**Pull request by jfboismenu**
Shotgun to ShotGrid. Branch is named incorrectly. Ticket is SG-22663.

### CODE REVIEW

File: tests/test_about_box.py [hunk 1]
```diff
@@ -72,19 +72,17 @@ def test_3rd_party_links(licence_file_links):
...
-        "https://github.com/shotgunsoftware/tk-framework-adobe/blob/master/software_credits",
-        "https://github.com/shotgunsoftware/tk-framework-desktopserver/blob/master/software_credits",
-        "https://github.com/shotgunsoftware/tk-framework-desktopstartup/blob/master/python/server/software_credits",
-        "https://github.com/shotgunsoftware/tk-framework-lmv/blob/master/software_credits",
-        "https://github.com/shotgunsoftware/tk-houdini/blob/master/software_credits",
-        "https://github.com/shotgunsoftware/tk-multi-launchapp/blob/master/software_credits",
-        "https://github.com/shotgunsoftware/tk-multi-pythonconsole/blob/master/software_credits",
-        "https://github.com/shotgunsoftware/tk-multi-reviewsubmission/blob/master/software_credits",
-        "https://github.com/shotgunsoftware/tk-nuke-quickreview/blob/master/software_credits",
+        "https://github.com/shotgunsoftware/python-api/tree/master/software_credits",
```
Comment (jfboismenu) [inline tests/test_about_box.py:75]: `/blob/master` is not a valid URL anymore!

File: python/tk_desktop/ui/about_screen.py [hunk 2]
```diff
@@ -71,10 +71,10 @@ def setupUi(self, AboutScreen):
         QtCore.QMetaObject.connectSlotsByName(AboutScreen)
 
     def retranslateUi(self, AboutScreen):
-        AboutScreen.setWindowTitle(QtGui.QApplication.translate("AboutScreen", "About Shotgun Desktop", None, QtGui.QApplication.UnicodeUTF8))
-        self.header.setText(QtGui.QApplication.translate("AboutScreen", "<b><big>Shotgun Desktop</big></b>", None, QtGui.QApplication.UnicodeUTF8))
+        AboutScreen.setWindowTitle(QtGui.QApplication.translate("AboutScreen", "About ShotGrid Desktop", None, QtGui.QApplication.UnicodeUTF8))
+        self.header.setText(QtGui.QApplication.translate("AboutScreen", "<b><big>ShotGrid Desktop</big></b>", None, QtGui.QApplication.UnicodeUTF8))
         self.body.setText(QtGui.QApplication.translate("AboutScreen", "Body", None, QtGui.QApplication.UnicodeUTF8))
-        self.copyright.setText(QtGui.QApplication.translate("AboutScreen", "Copyright ©2020 Shotgun Software Inc.\n"
```
Comment (willis102) [inline python/tk_desktop/ui/about_screen.py:77]: I noticed that you updated the copyright date when changed the string above -- do we want to update the copyright date here to 2021 as well?

File: resources/about_screen.ui [hunk 3]
```diff
@@ -113,7 +113,7 @@
       <string notr="true">font-size: 10px;</string>
      </property>
      <property name="text">
-      <string>Copyright ©2020 Shotgun Software Inc.
```
Comment (willis102) [inline resources/about_screen.ui:116]: And here as well?

**Review by willis102 — APPROVED** [review]
👍 LGTM!

File: python/tk_desktop/ui/about_screen.py (hunk 2 above)
Comment (jfboismenu) [reply python/tk_desktop/ui/about_screen.py:77]: Yup, I missed it. Thanks.

File: resources/about_screen.ui (hunk 3 above)
Comment (jfboismenu) [reply resources/about_screen.ui:116]: Thanks!

**Review by mathurf — APPROVED** [review]
Everything looks good!

**Pull request merged**